### ISSUE TITLE
ACC-282.1 (Level A) _ Texts,Buttons_ truncated and not visible 

### ISSUE THREAD

**Issue by Priya1Pavan** [kyma-project/busola#3259]
1.1 Internal Message Details
Internal Message:	
Test Plan:	ACC SAP Kyma runtime 2024
Message Priority:	Medium
Affected UI	labels
Affected Check Point:	ACC-282.1
Short text:	ACC-282.1 (Level A) _ Texts,Buttons_ truncated and not visible 

1.2 Application Details
Application Name	Kyma-dashboard
Technology:	SAP UI5 
Component:	
URL:	https://dashboard.stage.kyma.cloud.sap/cluster/garden-kyma-stage--c-7074971-external/kymamodules
Username/Password:	NA
1.3 Test Environment Details
Web browser:	Google Chrome Version 127.0.6533.89
Operating System:	Microsoft Windows 11 x64
Screen Reader:	JAWS 2024 Version 2024.2406.121 ILM

1.4 JIRA/SNOW Details
Issue:	Bug
Labels:	Accessibility

2 Screen Flow
•	Launch the application using given URL. 
•	Select the ‘Link to Kyma dashboard’ from the Kyma Environment section on your SAP BTP subaccount page. 
•	Go to KubeconfigURL:https://kyma-env-broker.cp.stage.kyma.cloud.sap/kubeconfig/DFEFE4E0-E202-4455-8A61-37511B4D7553
•	Click on  the link https://dashboard.stage.kyma.cloud.sap/clusters and select Connect.
•	Upload the kubeconfing file downloaded in the first step. You can either drag and drop it or select the upload icon to choose it from the device.
•	Next Step
•	

3. Issue Type: Manual
Observed Behavior:
On accessing the application in width equivalent to 320 CSS pixel ,The list items texts within the ‘Privacy’ and ‘Preview’ steps  and footer regions buttons are truncated and not visible completely.
Screenshot:
 
Screenshot 2:
 
Screenshot :default
 
Expected Behavior:

Truncation should be avoided, when the content of the screen is displayed on a width of 320 pixel
 
Content of a screen should be displayed and operable on a width of 320 CSS pixels without requiring two-dimensional scrolling,

Users with a small viewport, regardless of whether they use a small device or a considerable zoom, must be able to consume text as well as information given by other means without the need to scroll in reading direction while consuming one piece of information.
Having to scroll while reading disturbs the mental consumption of the information, is awkward and time consuming.
therefore the layout shall adapt to display information on restricted place without scrolling in reading direction.
[ACC-282.1 (Level A) _ Texts,Buttons_ truncated and not visible.docx](https://github.com/user-attachments/files/16727227/ACC-282.1.Level.A._.Texts.Buttons_.truncated.and.not.visible.docx)
[Issue processing-09.2021 (2).docx](https://github.com/user-attachments/files/16727228/Issue.processing-09.2021.2.docx)
